radicle/src/cob/patch.rs

```diff
@@ -818,9 +870,9 @@ impl<'a, 'g> Deref for PatchMut<'a, 'g> {
 #[derive(Default, Serialize)]
 #[serde(rename_all = "camelCase")]
 pub struct PatchCounts {
-    pub proposed: usize,
-    pub draft: usize,
-    pub archived: usize,
+    proposed: usize,
+    draft: usize,
+    archived: usize,
 }
 
```

Comment:
I wouldn't change `PatchCounts`, `CodeLocation`, `Merge` and `IssueCounts`, since they are immutable, and likely to remain that way.